Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -2594,12 +2678,23 @@ def __init__(self, outputs_prefix, outputs_key_field, outputs, indicators=None,
         self.raw_response = raw_response
         self.readable_output = readable_output
 
+        self.ignore_auto_extract = ignore_auto_extract
+
+    @staticmethod
+    def _get_timeline_category():
+        if IS_INTEGRATION:
+            return 'Integration Update'
```

Comment:
why it is important to even mention if it is `Integration Update` or `Automation Update`?

Reply:
That's the API from Server, @avidan-H can you elaborate on this?